pkg/pagerduty/pagerduty.go

```diff
@@ -23,6 +23,10 @@ const (
 	InvalidInputParamsErrorCode = 2001
 	// pagerDutyTimeout is the chosen timeout for api requests, can be changed later
 	pagerDutyTimeout = time.Second * 30
+	// CADEmailAddress is the email address for the 'Configuration-Anomaly-Detection' PagerDuty User
+	CADEmailAddress = "sd-sre-platform+pagerduty-configuration-anomaly-detection-agent@redhat.com"
```

Comment:
Does CAD have a config file? These values seem more suitable for that rather than hard-coded into the binary.

If not, I'm good with doing this for now

Reply:
No, CAD seems to use environment variables over config files which allows it to easily be configured at the cluster-level.

This isn't a value that should be altered between environments, so I think leaving it as a constant makes sense here.

Reply:
Thanks, sounds good to me